src/stock-adjustment-mobile/add-products-page/add-product-page.jsx

```diff
@@ -204,7 +211,7 @@ const AddProductsPage = ({}) => {
                                                 <SelectField
                                                     required
                                                     name={`${name}.product`}
-                                                    label="Product"
+                                                    label="Product*"
```

Comment:
Can we make these required stars red as in mockups?

Reply:
Yes. I can change this.

Reply:
I added this change with 'red asterisk'